go/worker/compute/executor/committee/node.go

```diff
@@ -894,6 +894,31 @@ func (n *Node) runtimeExecuteTxBatch(
 	inputRoot hash.Hash,
 	inputs transaction.RawBatch,
 ) (*protocol.RuntimeExecuteTxBatchResponse, error) {
+	// Ensure block round is synced to storage.
```

Comment:
This looks like a common operation (here and txpool), can it be extracted out?

Reply:
Extracted into a common method in the history interface.